app/forms/steps/claim_details_form.rb

```diff
@@ -0,0 +1,32 @@
+require 'steps/base_form_object'
+
+module Steps
+  class ClaimDetailsForm < Steps::BaseFormObject
+    BOOLEAN_FIELDS = %i[supplemental_claim preparation_time].freeze
+
+    attribute :prosecution_evidence, :string
+    attribute :defence_statement, :string
+    attribute :number_of_witnesses, :integer
+    attribute :time_spent_hours, :integer
+    attribute :time_spent_mins,  :integer
```

Comment:
any reason we don't have validations for the time fields?

Reply:
think it was because these fields were conditional - oversight - thanks